lib/internal/bootstrap_node.js

```diff
@@ -354,10 +352,9 @@
                                              originalConsole[key],
                                              wrappedConsole[key],
                                              config);
+      delete originalConsole[key];
```

Comment:
Why was this added and why were the two lines below removed?

Reply:
Ah, this should probably be a individual commit. It is only about a few entries but it did not feel good that we checked all entries again instead of just removing them beforehand.

Reply:
Hmm, I'm not sure what effect deleting `originalConsole[key]` could have. Is it really safe?